### ISSUE TITLE
SoftFX blitter unsupported 4k resolution

### ISSUE THREAD

**Issue by CorpseSlayer** [finalburnneo/FBNeo#1607]
Everytime i try to select  my resolution and try to go Fullscreen FBNeo SoftFX blitter give me error
it said: Problem Setting 3840x2160x16bpp and nothing changes from there, if i use other blitter
like experimental or DirectX 9 right now i am using Basic since SoftX is not working for me anyway to fix
this for 4k users?

<img width="795" alt="Screenshot 2023-12-03 091557" src="https://github.com/finalburnneo/FBNeo/assets/126618918/2a7e707b-7270-4db9-8d88-d2bc4acb7047">
![image_2023-12-03_092249013](https://github.com/finalburnneo/FBNeo/assets/126618918/3f64f591-81f4-43de-945e-5e15b4fc91d5)

**Comment by CorpseSlayer** [comment]
> directx 7 doesn't support 4k resolution. use directx 9.

yeah i would if it didn't brake some games like sfz3mix 0.22 not sure why but
if i select that blitter it will make the game look like a total mess this why i don't use it.

**Comment by barbudreadmon** [comment]
>i have a question if Direct 7 dont work on 4k why Basic Direct 7 works just fine?

The only thing i know is that it is known that directx 7 has a resolution limit : https://www.google.com/search?q=directx7+resolution+limit. I use neither windows nor 4k resolution so i wouldn't be able to tell anything beyond that.

>if i select that blitter it will make the game look like a total mess this why i don't use it.

Then you should write a report about that.
